src/llmq/quorums_instantsend.cpp

```diff
@@ -953,6 +953,10 @@ void CInstantSendManager::ProcessNewTransaction(const CTransactionRef& tx, const
         return;
     }
 
+    if (!masternodeSync.IsBlockchainSynced()) {
```

Comment:
Can't do this as it skips the `WriteInstantSendLockMined` calls in the case where you stopped your node and started it later (so not really IBD).

Reply:
makes sense